client/tinychain/ml/TEST_ADAM.py

```diff
@@ -0,0 +1,119 @@
+import math
+import numpy as np
+import operator
+import tinychain as tc
+import unittest
+
+from functools import reduce
+  
+import matplotlib.pyplot as plt
+import networkx as nx
+import tinychain as tc
+
+
+def visualize(state, cxt=tc.Context()):
```

Comment:
please add a `TODO` here to delete this code when after the debugger reaches its first release